R/string_operation.R

```diff
@@ -553,15 +553,20 @@ parse_number <- function(text, ...){
   } else if (!is.character(text)) {
     # non character data raises Error in parse_vector(x, col_number(), na = na, locale = locale, trim_ws = trim_ws) : is.character(x) is not TRUE
     # so explicitly convert it to character before calling readr::parse_number
-    as.numeric(readr::parse_number(as.character(text), ...))
+    # Passing non-ascii character causes an error so, remove non-ascii character before calling readr::parse_number.
+    # ref: https://github.com/tidyverse/readr/issues/1111
+    as.numeric(readr::parse_number(gsub('[^ -~]', '', as.character(text)), ...))
```

Comment:
Should be better to use [^\x20-\x7E] instead. It is hard to tell "[^ -~]" means non-ascii range .

Reply:
Thank you for the suggestion. I updated accordingly.